packages/calcite-components/src/components/accordion-item/accordion-item.scss

```diff
@@ -1,210 +1,202 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-accordion-border-color: Specify the border color of the component
+ * @prop --calcite-accordion-text-color: Specify the text color of the component
```

Comment:
I might expect these to be `--calcite-accordion-item-x` - the Accordion itself doesn't render any text.

Reply:
because an accordion-item only exists as a child of an accordion and these tokens can/should be applied at the parent level in most cases these make more sense from a devX perspective to be labeled as "accordion" not "accordion-item"